pvlib/irradiance.py

```diff
@@ -925,23 +920,41 @@ def reindl(surface_tilt, surface_azimuth, dhi, dni, ghi, dni_extra,
 
     Notes
     -----
-    The poa_sky_diffuse calculation is generated from the Loutzenhiser et al.
-    (2007) paper, equation 8. Note that I have removed the beam and ground
-    reflectance portion of the equation and this generates ONLY the diffuse
-    radiation from the sky and circumsolar, so the form of the equation
-    varies slightly from equation 8.
+    The Reindl (1990) model  for the sky diffuse irradiance,
+    :math:`I_d`, is as follows:
+
+    .. math::
+
+       I_{d} = DHI \left(A \cdot R_b + (1 - A)
+                         \left(\frac{1 + \cos\theta_T}{2}\right)
+       \left(1 + \sqrt{\frac{BHI}{GHI}} \sin^3(\theta_T/2)\right) \right).
+
+    :math:`DHI`, :math:`BHI`, and :math:`GHI` are the diffuse, beam (direct)
```

Comment:
Is "BHI" beam _horizontal_?

Reply:
Yes
[Context](https://github.com/pvlib/pvlib-python/pull/2193#issuecomment-2321944420) (was deciding between using what's in the paper vs pvlib user guide)

In the docs, the line "diffuse, beam (direct) and global horizontal irradiances" is supposed to mean diffuse horizontal irradiance, beam horizontal irradiance, and global horizontal irradiance. Do you think it could be made clearer?